test/663-combo-outdoor-landuse-pois.py

```diff
@@ -34,14 +34,12 @@
 assert_has_feature(
     16, 19314, 24677, 'pois',
     { 'kind': 'beach_resort', 'min_zoom': 16 })
-  
-#https://www.openstreetmap.org/node/1500943741
+
+# Way: Terrasol Beach Resort (381817391) https://www.openstreetmap.org/way/381817391
```

Comment:
I've been putting them all on the same line for other tests :( I'll start using this pattern going forward.